aspnetcore/tutorials/build-your-first-blazor-app.md

```diff
@@ -19,6 +19,15 @@ This tutorial shows you how to build and modify a Blazor app.
 
 Follow the guidance in the <xref:blazor/get-started> article to create a Blazor project for this tutorial. Name the project *ToDoList*.
 
+## Blazor app files
```

Comment:
This feels a bit haphazard. I think we want to keep the build your first Blazor app tutorial focused on helping the be successful at building their first app as quickly as possible. This content feels a bit haphazard and doesn't really fit in with that goal.

What if instead we created a new topic called "Project structure" that walks you through the files in a default Blazor project and explains how they all fit together? Something like this (but with a @guardrex touch): https://github.com/danroth27/IntroToBlazorWorkshop/blob/master/docs/02-project-structure.md